staging/src/k8s.io/apiserver/pkg/cel/environment/base.go

```diff
@@ -43,7 +44,14 @@ import (
 // desirable because it means that CEL expressions are portable across a wider range
 // of Kubernetes versions.
 func DefaultCompatibilityVersion() *version.Version {
-	return version.MajorMinor(1, 29)
+	ver := utilversion.DefaultEffectiveVersionRegistry.EffectiveVersionForOrRegister(
+		utilversion.ComponentGenericAPIServer, utilversion.K8sDefaultEffectiveVersion(),
+	).MinCompatibilityVersion()
+	// if MinCompatibilityVersion is not set for tests
+	if ver.Major() == 0 && ver.Minor() == 0 {
```

Comment:
This looks like it presents a risk of accidentally not specifying an important detail for using this library.    Can we go in the other direction and always set the MinCompatibilityVersion?

Reply:
done